.github/workflows/csi_operator_ci.yaml

```diff
@@ -1,13 +1,8 @@
-name: Operator test
-
-env:
-  OP_TEST_DEBUG: 1
-  OP_TEST_CONTAINER_OPT: "-t"
+name: Community Operators test
 on:
-  pull_request:
+  push:
```

Comment:
Why do we want to do it on `push` instead of PR?

Reply:
why not both?

Reply:
because the last code before the merge will be the same when it will be in `develop` so we will test the same thing twice

Reply:
unless there are missing changes from develop in the private branch...

Reply:
could we include all latest changes from develop in the beginning of the test?

Reply:
you would have to merge develop into your private branch before pushing to develop, no?

Reply:
it is not being enforced

Reply:
could we enforce it?

Reply:
added the trigger on push to develop and master